### DISCUSSION TITLE
Issue in running cnvkit.py diagram command

### DISCUSSION THREAD

**Discussion by korol996** [etal/cnvkit#871] [Q&A]
```shell
cnvkit.py diagram -s 002.Sample.cns 002.Sample.cnr
``` 
cnvkit.py diagram -s 002.Sample.cns 002.Sample.cnr
Traceback (most recent call last):
  File "/home/aclab/local/Python-3.12.1/bin/cnvkit.py", line 8, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/aclab/code/cnvkit/cnvlib/cnvkit.py", line 10, in main
    args.func(args)
  File "/home/aclab/code/cnvkit/cnvlib/commands.py", line 1365, in _cmd_diagram
    cnarr or segarr, args.sample_sex, args.male_reference, args.diploid_parx_genome
                                                           ^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'diploid_parx_genome'

![Screenshot from 2024-03-05 16-55-33](https://github.com/etal/cnvkit/assets/51869940/836b9e1d-3946-420b-90d8-848dbe9c1691)

**Comment by chendan17** [comment]
I have the same problem with you.
Do you have any solutions to solve it?
